packages/vscode-extension/src/handlers.ts

```diff
@@ -830,7 +830,11 @@ async function processResult(
 
   if (inputs?.env) {
     envProperty[TelemetryProperty.Env] = getHashedEnv(inputs.env);
-    envProperty[TelemetryProperty.AapId] = getTeamsAppIdByEnv(inputs.env);
+    const appInfo = getTeamsAppTelemetryInfoByEnv(inputs.env);
+    if (appInfo) {
+      envProperty[TelemetryProperty.AapId] = appInfo.appId;
```

Comment:
typo, and it looks like it was there before

Reply:
Yes. Fixed.